Integrations/AutofocusV2/AutofocusV2.py

```diff
@@ -774,6 +791,151 @@ def validate_no_multiple_indicators_for_search(arg_list):
     return
 
 
+def search_indicator(indicator_type, indicator_value):
+    headers = HEADERS
+    headers['apiKey'] = API_KEY
+
+    res = requests.request(
+        method='GET',
+        url=f'{BASE_URL}/tic?indicatorType={indicator_type}&indicatorValue={indicator_value}&includeTags=true',
```

Comment:
use `params` instead of adding them to the `url`:
```
params={
    'indicatorType': indicator_type,
    'indicatorValue': indicator_value,
    'includeTags': True,
}
```